data/human/hails.txt

```diff
@@ -1708,6 +1708,89 @@ phrase "friendly civilian"
 		", but I still had to take a big loan to buy my first ship."
 		", but it was my grandma's inheritance that allowed me to buy this ship."
 
+phrase "friendly civilian"
+	word
+		"Don't be too picky about the jobs you take. It's better to"
+	word
+		" "
+	word
+		"ship crooks, than having a crooked ship."
+		"ship slugs, than having a sluggish ship."
+		"ship bad bands, than having your ship badly banned."
```

Comment:
I'm not really sure what `badly banned` means here. If the captain doesn't have enough money, the ship gets banned?

Reply:
Ships don't get banned for having not enough money (might be a nice addition though; some rich planets that only allow the player to land if he/she has a sufficiently high net-worth), but I can think of a number of reasons why a captain would send a hail like this:
- Merchants are typically welcome everywhere, while military/army members could be "banned" from hostile planets. (So this hail having a meaning on that it is better to be a merchant shipping bad bands, than to be in the military and be unable to go to some places.)
- Bands and sports/competition games are quite popular in-lore. If a captain would seriously offend some band, then this could have a reputation impact and might make the captain unwelcome in some places.
- The captain might just be sending out nonsense that sounds cool.